bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -60,6 +61,7 @@ export default createContainer(() => {
     wasKicked: wasUserKicked(),
     isLoading: getLoading(),
     modal: getModal(),
+    getCaptionsStatus,
```

Comment:
That was the problem, once I put it this way (I initially tried) - it kicks me out of the client and doesn't let me back in.
No idea why. So I passed it as 'redirectToLogoutUrl' function below.